### DISCUSSION TITLE
馬斯克：Neuralink 大腦植入晶片會在 2022 年底進行首次人體試驗

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#569] [科技新知]
雖然腦機介面公司 Neuralink 創辦人馬斯克（Elon Musk），早在 2021 年就曾表示該公司在致力確保植入式腦機介面技術的安全性後，一直與美國食品藥物管理局（FDA）密切交流，以便在 2021 年展開首次的人體試驗。但直到現在，這件事和馬斯克過去許多承諾一樣又跳票了。但在馬斯克最近在 Twitter 上分享的一些更新訊息來看，他似乎不死心，並回應 Neuralink 會在 2022 年底進行首次人體試驗。

今年稍早之際，Neuralink 公開招聘臨床試驗主任及臨床試驗專員等職缺，此舉似乎也反應出馬斯克想要進行首次人體試驗的決心。最近，馬斯克在《TED》上接受專訪首度揭露，Neuralink 有可能解決病態性肥胖問題，並在未來 10 年內專注於解決大腦及脊椎損傷。身為神經介面專家的英國新堡大學教授指出，馬斯克近來有關 Neuralink 的聲明與主張是有可能實現的。而且 Neuralink 植入晶片比起其他病態性肥胖症的治療更不具侵入性。
https://technews.tw/2022/04/27/neuralink-test-in-2022/

**Comment by HuanYuuuuuu** [comment]
這些大腦植入晶片也因為其體積真的很小很小，之後植入的手術也會必須依靠機器來操作了

**Comment by awkward-willy** [comment]
很難想像一片晶片卡在大腦裡的感覺是什麼
不過若真的可行，或許能開啟一個全新的領域
不過這樣會不會對人體產生甚麼影響
也會是值得關注的議題

**Comment by Lai27330** [comment]
將晶片植入腦中感覺在目前接受度還不太高，而且透過影響神經的手段感覺容錯率非常低。

**Comment by Rosancl** [comment]
雖然植入的晶片本身很小，但目前來看接受度可能還是會比其他同樣植入醫療器材的治療小。

**Comment by yenwei12138** [comment]
如果真的像文章說的那麼小應該比較不會太害怕，倒是會比較害怕機器人幫自己動手術，不像真人來的有安全感

**Comment by treeann** [comment]
瑪斯克不是還把推特買下來 超誇張

**Comment by Yuhsin000** [comment]
印象中已經有將晶片植入大腦，讓身體有殘缺的人控制義肢的技術了

**Comment by liangyinglly** [comment]
很多影集與電影都有演過把晶片植入腦中的橋段，真的很想知道人類的細胞是如何跟晶片結合的，真的太神奇了

**Comment by Ben6uo** [comment]
感覺很神奇 什麼時候可以有第一次的人體鍊成呢?

**Comment by Jimmy911218** [comment]
這感覺有點侵犯了人體隱私了，而且可能在未來片可能會佔據人心

**Comment by sys511613** [comment]
科技在醫療領域有越來越多地方可以發揮了

**Comment by leon10456** [comment]
有點恐怖 我應該不會嘗試 但還是感謝樓主分享

**Comment by jcpan821** [comment]
以後真的可以用念力控制東西

**Comment by etin2468** [comment]
這個晶片雖然小，也說手術的侵入性低，但這種非生物性的東西植入身體感覺還是蠻怪的

**Comment by Hchejxnc** [comment]
感覺試驗成功，會開啟一個全新的領域

**Comment by ruthchen1020** [comment]
馬斯克真的很像外星人，但大腦確實是人類仍然最不熟悉的領域了，可能因為研究大腦對技術的要求太高了，但馬斯克可以做到。

**Comment by StevenWanglolz** [comment]
感覺這種科技很容易被人利用ㄟ，像是一把雙面刃!

**Comment by CPCpyt** [comment]
感覺在未來普及後，有如同科幻小說世界也將離我們不遠。

**Comment by Doreen331** [comment]
將晶片植入大腦讓我想起一部之前看到的影集「黑鏡」，其中一集探討的內容就是將晶片植入人類的大腦，雖然科技進步確實可以幫助我們的醫療技術，但若是像是植入晶片這中需要將一個實體長期置入體內的應用，我認為還是不太適合

**Comment by Floriaand** [comment]
希望會成功，未來可以治療更多疾病

**Comment by Evan0979** [comment]
感覺還是很可怕呀,以後會不會成為一些極權政府控制人民的工具啊

**Comment by harrylee1971** [comment]
以後腦袋中有一枚晶片，讓眼睛內看到AR的世界或許也不錯，肯定又是革命性的發展

**Comment by tower20021202** [comment]
感覺挺酷的，期待他的發展
